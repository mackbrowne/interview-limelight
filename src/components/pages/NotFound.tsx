// Framework
import React from 'react';

// Components
import { Container, Row, Col } from 'react-bootstrap';

const NotFound = () => (
  <Container className="not-found">
    <Row>
      <Col>
        <p>
          <strong>Error [404]</strong>: {window.location.pathname} does not
          exist.
        </p>
      </Col>
    </Row>
  </Container>
);

export default NotFound;
