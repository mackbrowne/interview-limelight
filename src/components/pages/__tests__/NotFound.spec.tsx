import React from 'react';
import { mount } from 'enzyme';

import NotFound from '../NotFound';

describe('<NotFound /> Snapshot', () => {
  it('renders snapshot with no props', () => {
    const wrapper = mount(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
