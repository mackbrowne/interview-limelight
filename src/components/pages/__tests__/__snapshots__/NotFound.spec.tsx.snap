// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Snapshot renders snapshot with no props 1`] = `ReactWrapper {}`;
